Documentation/subcommands/api-service.md

```diff
@@ -17,3 +17,21 @@ Specify the address `0.0.0.0` to listen on all interfaces.
 
 The interfaces are defined in the [protobuf here](../../api/v1alpha/api.proto).
 Here is a small [Go program](../../api/v1alpha/client_example.go) that illustrates how to use the API service.
+
+## Options
+
+| Flag | Default | Options | Description |
+| --- | --- | --- | --- |
+| `--listen` |  `localhost:15441` | An address to listen on | Address to listen on client API requests |
+
+## Global options
```

Comment:
Perhaps we could just add a reference to the global options.  We're going to need to change each file whenever a change is made to global options.